demo/Makefile

```diff
@@ -0,0 +1,78 @@
+
+# Domain to use when starting flynn
+CLUSTER_DOMAIN := demo.localflynn.com
+CLUSTER := default
+BOOTSTRAP_LOG_FILE := bootstrap.${CLUSTER}.log
+
+# Providers: virtualbox, vmware_fusion, aws
+PROVIDER := virtualbox
+
+
+.PHONY: install init bootstrap up down halt update ssh reload reset token destroy dashboard ssh-cmd
+
+
+# Install flynn cli on host
+install:
+	L=/usr/local/bin/flynn && curl -sL -A "`uname -sp`" https://dl.flynn.io/cli | zcat >$L && chmod +x $L
+
+# Initialize a new cluster
+init:
+	@$(MAKE) up
+	@$(MAKE) bootstrap
+	flynn key add
+
+# Bootstrap the cluster
+# This should only be called once per cluster
+# Runs bootstrap in VM and cluster add on host
+bootstrap:
+	@$(MAKE) CMD='CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" flynn-host bootstrap /etc/flynn/bootstrap-manifest.json 2>&1' ssh-cmd | tee ${BOOTSTRAP_LOG_FILE}
+	cat ${BOOTSTRAP_LOG_FILE} | awk '/flynn cluster add/' | awk 'BEGIN {RS=""}{gsub(/\r/,"",$$0); print $$0}' | sh -s
+	rm ${BOOTSTRAP_LOG_FILE}
+
+# Start the VM and flynn
+up:
+	CLUSTER_DOMAIN="${CLUSTER_DOMAIN}" vagrant up --provider ${PROVIDER}
+	@$(MAKE) CMD='sudo start flynn-host || echo flynn-host already running or unable to start' ssh-cmd
+
+# Stop flynn
+down:
+	@$(MAKE) CMD='sudo stop flynn-host' ssh-cmd
+
+# Halt the VM
+halt:
+	vagrant halt
+
+# Update flynn in the VM
+# VM must already be up and running
+update:
+	@$(MAKE) CMD='sudo bash < <(curl -fsSL https://dl.flynn.io/install-flynn) && sudo flynn-host update' ssh-cmd
```

Comment:
I might have misunderstood your comment @titanous. Did you mean `make down` is unnecessary or the `start flynn-host` in `make up` is unnecessary?